### ISSUE TITLE
Neovim Support

### ISSUE THREAD

**Issue by rperryng** [jesseleite/vim-agriculture#4]
Hey there.  I recently updated `vim-agriculture` and started getting these errors.

```
Error detected while processing function agriculture#fzf_rg_raw:
line    5:
E117: Unknown function: trim
E15: Invalid expression: 'rg --column --line-number --no-heading --color=always ' . trim(userOptions . ' ' . a:command_suffix)
line    6:
E121: Undefined variable: command
E116: Invalid arguments for function extend
E116: Invalid arguments for function call
E15: Invalid expression: call('fzf#vim#grep', extend([command, 1], a:000))
```

I believe it's because Neovim doesn't implement the `trim()` function.  It looks like this was added to Vim in version [8.0.1630](https://stackoverflow.com/a/4479072/11600440).  Is there any plan to support Neovim due to this API difference?  If not, I understand, and will happily downgrade `vim-agriculture` back to a commit before this change was introduced (or, if I'm feeling really inspired perhaps make a PR with your input on how to workaround this 😄).

Thanks again for this plugin!  It's one I find immensely useful.  Have a great day ☀️

**Comment by jesseleite** [comment]
Thank you for kind words!  Oh interesting, I will look into it when I can for sure!  You're welcome to PR if you feel comfortable.

My initial thoughts are, if `trim()` isn't available in neovim, there's got to be similar functionality in neovim.  If necessary, could probably wrap it with our own trim function, and do a vim/neovim check within.

**Comment by jesseleite** [comment]
Fixed by #5 🎉
